File: Templates/Day_1/CalculatingPi/Fortran/pi_square_serial.f90
Program pi

  implicit none

  integer, parameter :: finval = 10000
  double precision :: pi_square = 0.0d0
  double precision :: factor
  integer:: i
  
  do i= 1, finval
     factor = i
     pi_square = pi_square + 1.0d0/(factor * factor)
  enddo

  print *, 'Pi**2 =', 6.0d0 * pi_square
  
end Program pi

File: Templates/Day_1/CalculatingPi/C/pi_square_serial.c
#include <stdio.h>

int main()
{

  const int finval = 1000000000;

  double pi_square = 0.0;

  for ( int i=1 ; i <= finval; i++)
    {  
      double factor = (double) i;
      pi_square += 1.0/( factor * factor );
    }

  printf ( "Pi^2 = %.10f\n", pi_square*6.0);

  return 0;
}
